composer.lock

```diff
@@ -1132,7 +1132,50 @@
                 }
             ],
             "time": "2023-08-28T11:09:02+00:00"
-        }
+        },
+	    {
+		    "name": "firebase/php-jwt",
+		    "version": "v6.10.0",
```

Comment:
Please also commit the changes to composer.json or the package will be remove next time a full dependency update is made.

Reply:
sorry, forgot for this.
changed composer.json